### DISCUSSION TITLE
How do I specify the chat format for the phi3.5 model? It seems like no one knows

### DISCUSSION THREAD

**Discussion by suhyun01150** [abetlen/llama-cpp-python#1729] [Q&A]
After setting chat_format to None, can I execute phi3, gemma, etc., exactly as described in the README?

llm.create_chat_completion(
    messages = [
        {"role": "system", "content": "You are an assistant who perfectly describes images."},
        {
            "role": "user",
            "content": "Describe this image in detail please."
        }
    ]
)

If it works, how does it do so? When I load the model, I see an output like this. Does it automatically recognize the format?
tokenizer.chat_template str              = {% for message in messages %}{% if me...

somebody help me~~

**Comment by controlecidadao** [comment]
Hello!

I have been using it without specifying the prompt format and have obtained interesting results.
Check out our **Samantha Interface Assistant** project and take a look at the code: 

https://github.com/controlecidadao/samantha_ia

<br>

This video shows **Phi 3.5** *vs* **Gemma 2** in an intelligence challenge about human nature using the interface. **Llama 3.1** plays the role of judge:

https://www.youtube.com/watch?v=KgicCGMSygU
